### ISSUE TITLE
Mac OS User Interface issues

### ISSUE THREAD

**Issue by Bdom25** [vergecurrency/VERGE#190]
Hi there,

First up, excellent job on the release!

Just thought i'd post a few UI issues i've noticed with this release. Nothing major and easily fixable.

<img width="179" alt="1" src="https://user-images.githubusercontent.com/34997675/34466551-65090020-ef1b-11e7-8e56-eace5f4d74f2.png">

Image2 1 is when you right click the icon on the dock. Shortcut text for the sending and receiving window in the software is blank.

<img width="643" alt="2" src="https://user-images.githubusercontent.com/34997675/34466562-8dc8a786-ef1b-11e7-87c0-ec53bd8444a3.png">

Image 2

Opening the preferences dialogue shows the unit measurement screen but the text / drop down box is cut off and the drop down box has a bug (difficult to see everything, looks sloppy and difficult to use)

<img width="648" alt="3" src="https://user-images.githubusercontent.com/34997675/34466570-ce02aa0e-ef1b-11e7-98f5-5e45fe2aadc0.png">

<img width="658" alt="4" src="https://user-images.githubusercontent.com/34997675/34466575-d8100f50-ef1b-11e7-9c3a-0ee11c3c41ea.png">

Image 3 & 4

Same error as image 2, dropdown boxes are buggy and dont display text or work correctly.

<img width="909" alt="5" src="https://user-images.githubusercontent.com/34997675/34466579-ee73656c-ef1b-11e7-89b5-86b53f1e61c7.png">

Image 5

Text top and bottom are cut off, might be good to have this text have its own white BG rather than the background image used as it makes it hard to read.

None of this is super serious but does look a little sloppy, I see its still "beta" in the version dialogue so I guess its to be expected but not sure what other users might expect!

Again, good work!

**Comment by gutterstyle** [comment]
posted in discord that this didnt work right and got insta kicked from verge discord. was stoked for verge but community is like toxic children.

**Comment by DarthJahus** [comment]
@gutterstyle, try to understand that everyone is on the edge, actually. Most moderators didn't sleep and sacrified their new year's eve for the community. In this situation, they might have the hand on the trigger and easily use the ban hammer, dealing with tons of trolls and immature people.

**Comment by Mathematixx** [comment]
I can't get the wallet to open in macOS Sierra 10.12.6
How did you go about the process?

**Comment by DarthJahus** [comment]
@Mathematixx, the issue is known and is being fixed. Keep calm and get some popcorn 😄 🍿 The devs are taking care of us.

**Comment by corecrypto** [comment]
Sure.

Bobby Clint Baldwin
CEO
Core CryptoCurrency

**Comment by jeffreyrufino** [comment]
So this is what a 2,285,205,800 USD  wallet looks like? hahaha

**Comment by DarthJahus** [comment]
@jeffreyrufino, ever seen Bitcoin Core (224 B$ MC) Qt wallet? Bitcoin Gold (4 B$ MC) Qt wallet? You should take a look.

**Comment by BiggJimmy** [comment]
Unfortunately, there are substantial amount of errors/ gui displacement that need to be ironed it.. I think this project has been rushed 100% with pressure and rapid growth of the verge community. I think a new fresh approach needs to be established to iron out these errors, if it is to be successfull.

**Comment by vorlonempire** [comment]
![windows_blank_bug](https://user-images.githubusercontent.com/7526868/34469103-e1ef31ee-ef17-11e7-862f-68ea1b4ed4c7.png)

The same issue appear on windows QT build

**Comment by dbtftw** [comment]
yeah was gonna file and issue but this is the master of the Dupes. Tracking here.

**Comment by corymsmith** [comment]
I’ve fixed part of the issue in image 2 locally (the box being cut off), just haven’t had a chance to sort out why the drop downs are so damn buggy. Anybody have a clue? Hoping to submit a PR tonight / tomorrow AM
